src/site/content/en/blog/pwa-url-handler/index.md

```diff
@@ -0,0 +1,291 @@
+---
+layout: post
+title: PWAs as URL Handlers
+subhead: |
+  Let installed PWAs handle URLs for a more integrated experience.
+authors:
+  - thomassteiner
+date: 2021-06-01
+# updated:
+description: |
+  After registering a PWA as a URL handler, when a user clicks on a hyperlink that matches
+  one of the registered URL patterns, the registered PWA will open.
+hero: image/8WbTDNrhLsU0El80frMBGE4eMCD3/EjFQNwv2IfINKykzsxxs.png
+alt: |
+  The Windows Skype instant messenger app next to the installed demo PWA, which is opened
+  in standalone mode after clicking a link handled by it in a Skype chat message.
+tags:
+  - blog # blog is a required tag for the article to show up in the blog.
+  - capabilities
+  - progressive-web-apps
+---
+
+{% Aside %} PWAs as URL Handlers is part of the [capabilities project](https://web.dev/fugu-status/)
+and is currently in development. This post will be updated as the implementation progresses.
+{% endAside %}
+
+## What is PWAs as URL Handlers? {: #what }
+
+Imagine you chat with a friend using an instant messenger application like Messages on macOS and you
+talk about music. Further imagine you both have the `music.example.com` PWA installed on your
+devices. If you want to share your favorite track for your friend to enjoy, you can send them a deep
+link like `https://music.example.com/rick-astley/never-gonna-give-you-up`. Since this link is pretty
+long, the developers of `music.example.com` may have decided to add an additional short link to each
+track, like, for example, `https://🎵.example.com/r-a/n-g-g-y-u`.
+
+PWA as URL Handlers is a proposal that allows apps like `music.example.com` to register themselves
+as URL handlers for URLs that match patterns like `music.example.com`, `*.music.example.com`, or
+`🎵.example.com`, so that links from outside of the PWA, for example, from an instant messenger
+application or an email client, open in the installed PWA rather than in a browser tab.
+
+The PWA as URL Handlers proposal consists of two additions:
+
+1. The `"url_handlers"` Web App Manifest member.
+1. The `web-app-origin-association` file format for validating out-of-scope URL associations.
+
+### Suggested use cases for PWAs as URL Handlers {: #use-cases }
+
+Examples of sites that may use this API include:
+
+- Music or video streaming sites so track links or playlist links open in the player experience of
```

Comment:
>player experience

You mean the view of the app lacking browser chrome, right? I think I inferred it because I work on Fugu all the time. Is there a better way to say this?

Reply:
Not necessarily, but this chrome-less view is one way.